content/en/docs/usage/install.md

```diff
@@ -153,7 +153,7 @@ too by specifying the install location of the repository from
 [GitHub](https://github.com/unikraft/kraft.git):
 
 ```bash
```

Comment:
Use `console` instead of `bash`.

Reply:
@mkroening , I would still go for using `@stable`. We would need to update the `stable` branch on the [`kraft` repository](https://github.com/unikraft/kraft). @nderjung , could you take care of that?

But, I would add a mention such as: "If you want to use the latest features of `kraft` use this command: `... @staging`".

Reply:
Both done. 👍